arrow-libs/optics/arrow-optics-test/src/commonTest/kotlin/arrow/optics/DSLTest.kt

```diff
@@ -0,0 +1,106 @@
+//package arrow.optics.arrow.optics
```

Comment:
This file can be deleted, it's duplicated with `DSLTest.kt` inside `arrow-optics/test`